src/Forms/Form/Form.stories.mdx

```diff
@@ -1,336 +1,545 @@
+					options: [
+						{
+							label: 'Option1',
+							value: 'value1',
+						}
+					]
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+
+## FormFieldChipSingleSelect
+- The `FormFieldChipSingleSelect` component is built over a wrapper for [Material-UI Autocomplete](https://material-ui.com/components/autocomplete/) but with SimpleView brand colors.
+- [**Playground**](/?path=/story/formfields-formfieldchipsingleselect--playground)
+- Default Value: `string` - Value of the selected option.
+
+### Props (Input Settings)
+* **options** - `array` of `{ label: string, value: string, selected?: boolean }` required - Array of options to be rendered containing their corresponding label, value, and indicator of whether its selected or not.
+* **onSelect** - `(...args) => void` optional - Callback to be executed when an option gets selected.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'chip',
+				inputSettings: {
+					options: [
+						{
+							label: 'Option1',
+							value: 'value1',
+							selected: false,
+						}
+					]
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+
+## FormFieldColorPicker
+- The color picker component allows users to select from pre-defined colors (swatches) or custom colors using a HSB selection interface.
+- The implementation of this component is based on the following package: [React Color](https://casesandberg.github.io/react-color/)
+- [**Playground**](/?path=/story/formfields-formfieldcolorpicker--playground)
+- Default Value: `ColorResult` - Object containing all the different variants for the color selected.
+
+### ColorResult (Type)
+* **hex** - `string`
+* **hsl** - `object`
+	* **a** - `number | undefined`
+	* **h** - `number`
+	* **s** - `number`
+	* **l** - `number`
+* **rgb** - `object`
+	* **a** - `number | undefined`
+	* **r** - `number`
+	* **g** - `number`
+	* **b** - `number`
+
+### Props (Input Settings)
+This component doesn't require any additional props.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'color',
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+
+## FormFieldDateField
+- It allows the user to view and pick dates from a calendar widget or manually type the date in the text field.
+- [**Playground**](/?path=/story/formfields-formfielddatefield--playground)
+- Default Value: `string` - Date in UTC format transformed to string.
+
+### Props (Input Settings)
+* **showTime** - `boolean` optional - When true a time field will appear next to the date field.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'date',
+				inputSettings: {
+					showTime: false,
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+### FormFieldDropdownSingleSelection

```

Comment:
Let's make all titles one level smaller, for example ## FormFieldDropdown, and props would be ### Props